src/components/AffectedOfferings.vue

```diff
@@ -102,6 +112,14 @@ function moduleComponentName(moduleName: string = '<module not set>', componentN
   <div v-if="theAffects" class="osim-affects">
     <h4 class="mb-4">
       Affected Offerings
+      <button
+        v-if="areAllModulesCollapsed"
```

Comment:
I feel like this conditional should be changed, e.g. if there are 15 modules and only one is expanded, it should be possible to expand the other 14 without having to manually collapse the one expanded.

Same way the https://github.com/RedHatProductSecurity/osim/pull/239/files#diff-14aca78bde51d2bb8135088852aa914ecb6c5fc2d0033143a4299ddeb20d73beL105-L112 function works, instead of checking `areAllModulesCollapsed`, check `isAnythingCollapsed`

Reply:
Good catch, thanks for pointing out. Rechecking it I think the actual code is Ok, the problem is on the name  `areAllModulesCollapsed`, it is wrong and lead to confusions (see next comment).